eureka-client/build.gradle

```diff
@@ -12,7 +12,10 @@ dependencies {
     api 'jakarta.ws.rs:jakarta.ws.rs-api:3.0.0'
     api 'jakarta.inject:jakarta.inject-api:2.0.1'
     api 'jakarta.annotation:jakarta.annotation-api:2.1.0'
-    api "com.netflix.servo:servo-core:${servoVersion}"
+    api "com.netflix.spectator:spectator-api:${spectatorVersion}"
+    api "org.slf4j:slf4j-api:${slf4jVersion}"
+    implementation "com.netflix.netflix-commons:netflix-eventbus:0.3.0"
```

Comment:
`netflix-commons` is also legacy stuff that brings in dependencies that can be problematic, e.g. [servlet-api](https://github.com/Netflix/netflix-commons/pull/24).

Reply:
That looks like an issue with the cherry pick from my old commit, this is already present in the build.gradle, I agree netflix-common is toxic but for now we'll leave that for future pull request 😄

Reply:
I changed it from api->implementation which should help a little bit for now